regulations/templates/regulations/section.html

```diff
@@ -23,7 +23,16 @@
 
 <div class="box">
 
-    <aside class="left-sidebar">
+    <aside class="left-sidebar" data-state="expanded">
```

Comment:
@inkandsound what's your opinion on this experiment using `data-state` to describe this behavior. I'm aiming to make the intent (and the probable implementation) more immediately apparent both here and in the js.

Reply:
@whytheplatypus I like that!